_search-plugins/search-pipelines/index.md

```diff
@@ -0,0 +1,387 @@
+---
+layout: default
+title: Search pipelines
+nav_order: 100
+has_children: true
+has_toc: false
+---
+
+# Search pipelines
+
+Search pipelines is an experimental feature. Therefore, we do not recommend the use of search pipelines in a production environment. For updates on the progress of search pipelines, or if you want to leave feedback that could help improve the feature, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/6278).    
+{: .warning}
+
+To integrate result rerankers, query rewriters, and other components that operate on queries or results, you can use _search pipelines_. Search pipelines make it easier for you to process search queries and search results right in OpenSearch, as opposed to building plugins or creating additional logic elsewhere in your application. Thus, you can customize search results without adding complexity to your application. Search pipelines also provide a mechanism for modularizing your processors so you can add and reorder them easily. 
```

Comment:
Can we say something that indicates that it's easy to build these and not just integrate? So maybe "...build or integrate result rerankers...?" Also, I think instead of saying "...as opposed to building plugins or creating additional logic elsewhere in your application..." we want to say that building processors could reduce complexity in building plugins used in search requests by allowing the functionality of a plugin to be composed by multiple processors. Might be too wordy, but search processors are not there to replace plugins.